From 79b38b9f85da868ca59b66715c20aa55104b640b Mon Sep 17 00:00:00 2001
From: Robert Marko <robert.marko@sartura.hr>
Date: Fri, 2 Oct 2020 10:43:26 +0200
Subject: [PATCH] arm: dts: ipq4019: QCA807x properties

This adds necessary DT properties for QCA807x PHY-s to IPQ4019 DTSI.

Signed-off-by: Robert Marko <robert.marko@sartura.hr>
---
 arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

--- a/arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi
+++ b/arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi
@@ -758,6 +758,10 @@
 				ethphy4: ethernet-phy@4 {
 					reg = <4>;
 				};
+
+				psgmiiphy: psgmii-phy@5 {
+					reg = <5>;
+				};
 			};
 		};
 
